src/Sulu/Bundle/MediaBundle/Tests/Fixtures/image/formats/current.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do not name these files current, but use the version of the format somehow. Then we don't have to rename everything when we change something again.